server/server.js

```diff
@@ -33,5 +44,13 @@ server.listen(config.port, config.ip, function () {
   console.log('Express server listening on %d, in %s mode', config.port, app.get('env'));
 });
 
+
+// Init database
+
+
+app.use(passport.initialize());
+app.use(passport.session());
```

Comment:
Based on how we set up the server and config files before, I think the middleware features for Express should be defined in config/express.js. You can also remove any duplicated `require()` statements from this file if they're going to be used in express.js.